src/scoreCard.js

```diff
@@ -0,0 +1,59 @@
+var ScoreCard = function() {
+    this.frames = [new Frame(), new Frame(), new Frame(), new Frame(), new Frame(),
+                   new Frame(), new Frame(), new Frame(), new Frame(), new Frame(), ]
+    this.currentFrame = 0
+}
+
+ScoreCard.prototype.roll = function(pins) {
```

Comment:
This function has too many statements. (15)<br>This function's cyclomatic complexity is too high. (9)